src/snyk/cli/services/cliService.ts

```diff
@@ -64,12 +64,17 @@ export abstract class CliService<CliResult> extends AnalysisStatusProvider {
       if (!killed) this.logger.error('Failed to kill an already running CLI instance.');
     }
 
+    const foldersToTest = this.workspace.getWorkspaceFolders();
+    if (foldersToTest.length == 0) {
+      throw new Error('No workspace was opened.');
+    }
+
     this.cliProcess = new CliProcess(this.logger, this.config);
-    const args = this.buildArguments();
+    const args = this.buildArguments(foldersToTest);
 
     let output: string;
     try {
-      output = await this.cliProcess.spawn(cliPath, args);
+      output = await this.cliProcess.spawn(cliPath, foldersToTest[0], args);
```

Comment:
Why limit this to one folder and not loop over all?

Reply:
Would you spawn multiple CLI processes with different cwds?

Reply:
yes!

Reply:
That would be an option but would require some rework around how we handle reports from multiple CLI instances. Since there's no actual need for that at the moment, I would not be addressing it in this PR.